docs/source/virtual_ecosystem/implementation/abiotic_simple_implementation.md

```diff
@@ -1 +1,140 @@
-# Simple implementation of the abiotic environment
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: pyrealm_python3
+  language: python
+  name: pyrealm_python3
+---
+
+# Implementation of the abiotic simple model
+
+This section walks through the steps in generating and updating the
+[abiotic_simple](../../../../virtual_ecosystem/models/abiotic_simple/abiotic_simple_model.py)
+model which is currently the default abiotic model version in the Virtual Ecosystem
+configuration.
+
+The `abiotic_simple` model is a simple regression model that estimates microclimatic
+variables based on empirical relationships between leaf area index (LAI) and atmospheric
+temperature (T), relative humidity (RH), and vapour pressure deficit (VPD) to derive
+logarithmic profiles of these variables from external climate data such as regional
+climate models or satellite observations. The model also provides information on
+atmospheric pressure and $\ce{CO_{2}}$ and soil temperatures at different depths.
+
+## Required input data
+
+The `abiotic_simple` model requires a timeseries of the following variables to
+initialise and update the model (`_ref` refers to the referenece height 2 m above the
+canopy):
+
+``` python
+required_init_vars=(
+        "air_temperature_ref", # [C]
+        "relative_humidity_ref", # [-]
+    ),
+
+required_update_vars=(
+        "air_temperature_ref", # [C]
+        "relative_humidity_ref", # [-]
+        "vapour_pressure_deficit_ref", # [kPa], calculated in __init__
+        "atmospheric_pressure_ref", # [kPa]
+        "atmospheric_co2_ref", # [ppm]
+        "leaf_area_index", # [m m-1], provided by plants model
+        "layer_heights", # [m], provided by model core structure
+    )
+```
```

Comment:
I think we should avoid having this as code - it's a bit scary at this stage in the docs.  I think we can probably come up with a code recipe for automatically generating a prettily printed Variables block to include here, probably using @dalonsoa's variable system.